README.md

```diff
@@ -55,83 +58,122 @@ RelBench has the following main components:
 # Installation
 
 You can install RelBench using `pip`:
-
-```
+```bash
 pip install relbench
 ```
 
-This will allow usage of the RelBench data and task loading functionality. To additionally use the example GNN scripts in the ```examples``` directory, and the graph-related helper functions found in ```relbench/modeling``` it is also necessary to install [PyTorch Geometric](https://github.com/pyg-team/pytorch_geometric) and [PyTorch Frame](https://github.com/pyg-team/pytorch-frame). PyTorch Frame can simply be installed with
+This will allow usage of the core RelBench data and task loading functionality.
 
+To additionally use `relbench.modeling`, which requires [PyTorch](https://pytorch.org/), [PyTorch Geometric](https://github.com/pyg-team/pytorch_geometric) and [PyTorch Frame](https://github.com/pyg-team/pytorch-frame), install these dependencies manually or do:
 
+```bash
+pip install relbench[full]
 ```
-pip install pytorch_frame
+
+
+To run the example GNN scripts in the `examples` directory, use:
+```bash
+pip install relbench[example]
```

Comment:
Should we also include after this an example of how to use a GNN script?

Reply:
Done!